### PR TITLE
[SEPOLICY] Fix few denials for cnd

### PR DESCRIPTION
**Pull request by Haxk20**
Allow cnd to create sockets and allow cnd to send data to sockets.

### CODE REVIEW

**Comment by jerpelea** [comment]
@Myself5 @MarijnS95

**Comment by Haxk20** [comment]
I think you meant @ix5

**Comment by Haxk20** [comment]
@ix5 please review.

**Comment by MarijnS95** [comment]
As said before, check the readme for contribution guidelines and submit the patch in a way that is consistent with other commits.

And don't forget to resolve the conflict.

File: vendor/cnd.te [hunk 1]
```diff
@@ -2,3 +2,4 @@ type cnd, domain;
 type cnd_exec, exec_type, vendor_file_type, file_type;
 
 init_daemon_domain(cnd)
+allow cnd self:udp_socket create_socket_perms;
```
Comment (ix5) [inline vendor/cnd.te:5]: Newline before this line please.

**Comment by Haxk20** [comment]
@ix5 @MarijnS95

**Comment by ix5** [comment]
LGTM

**Comment by MarijnS95** [comment]
Thanks for adding the denials, looks good now. Note it's common to indent these in commit messages with 4 spaces, to indicate a "code block" rather than plain text.

Really nitpicky, shouldn't `vendor/hal_wifi_supplicant_default.te` remain sorted?

**Pull request merged**